packages/devtools-network-console/src/ui/generic/BoolCheck.tsx

```diff
@@ -3,14 +3,30 @@
 
 import * as React from 'react';
 
+import { getText, ILocalized, LocalizationConsumer } from 'utility/loc-context';
+
 export interface IBoolCheckProps {
     isChecked: boolean;
 }
 
-export default function BoolCheck(props: IBoolCheckProps) {
+function BoolCheck(props: IBoolCheckProps & ILocalized) {
+    const label = props.isChecked ? 
+        getText('BoolCheck.yesLabel', props) :
+        getText('BoolCheck.noLabel', props);
+
     if (props.isChecked) {
-        return <span aria-label="yes">✓</span>;
+        return <span aria-label={label}>✓</span>;
```

Comment:
nit, should we consider adding aria-checked? or is this not interactive?

Reply:
It's not interactive.